### ISSUE TITLE
No destination URL shown in bottom status bar on mouse-hover on images (including `Brave` `Ads`)

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#35233]
## Description 

No destination URL shown in bottom status bar on mouse-hover on images (including `Brave` `Ads`)

## Steps to Reproduce

   1. install `1.63.106`
   2. launch Brave
   3. open a new-tab page
   4. scroll down
   5. click on `Turn on Brave News`
   6. mouse-hover over any image

## Actual result:

No destination URL is shown on on mouse-hover on the images

<img width="1976" alt="Screenshot 2024-01-09 at 11 36 26 PM" src="https://github.com/brave/brave-browser/assets/387249/c801ce40-27e0-437c-a6b8-2a284af9e15c">

## Expected result:

Per `1.61.114` parity, we should show the destination URL for images, too (including `Brave` `Ads`)

<img width="1976" alt="Screenshot 2024-01-09 at 11 36 31 PM" src="https://github.com/brave/brave-browser/assets/387249/cec6d078-04ff-4de0-a508-ccbb7fdf5bf7">

## Reproduces how often: 

100%

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc @LorenzoMinto @fallaciousreasoning @mattmcalister @brave/qa-team

**Comment by LorenzoMinto** [comment]
Is this a duplicate of https://github.com/brave/brave-browser/issues/35234? If so we can close this.

**Comment by stephendonner** [comment]
> Is this a duplicate of #35234? If so we can close this.

No, that one is about the feed source being shown as a titletip; this is about just showing the URL we're already showing for the text/summary portion of the card, but not its images (or `Ads`, as noted).
